src/shogun/regression/LeastAngleRegression.h

```diff
@@ -72,14 +73,19 @@ class CFeatures;
 class CLeastAngleRegression: public CLinearMachine
 {
 public:
+	
+	/*const unsigned int Lower = 1;
```

Comment:
Left over when I was attempting to template it the train method, forgot to remove.